trestle/tasks/cis_to_catalog.py

```diff
@@ -0,0 +1,291 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2021 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""OSCAL transformation tasks."""
+
+import configparser
+import datetime
+import logging
+import pathlib
+import traceback
+import uuid
+from typing import List, Optional, ValuesView
+
+import trestle
+from trestle.core import const
+from trestle.oscal import OSCAL_VERSION
+from trestle.oscal.catalog import Catalog
+from trestle.oscal.catalog import Control
+from trestle.oscal.catalog import Group
+from trestle.oscal.common import Link
+from trestle.oscal.common import Metadata
+from trestle.tasks.base_task import TaskBase
+from trestle.tasks.base_task import TaskOutcome
+
+logger = logging.getLogger(__name__)
+
+
+class Node():
+    """Representation of CIS profile entry."""
+
+    def __init__(self, name=None, description=None):
+        """Initialize node."""
+        self._name = name
+        self._description = description
+
+    @property
+    def name(self) -> str:
+        """Node name."""
+        return self._name
+
+    @property
+    def description(self) -> str:
+        """Node description."""
+        return self._description
```

Comment:
For something like this why don't we just use a pydantic class (off of BaseModel not OscalBaseModel)?

Reply:
Fixed.